setup.py

```diff
@@ -36,6 +36,11 @@
     packages=find_packages(exclude=("test",)),
     include_package_data=True,
     install_requires=[
+        "Click==8.0.1",
+        "dice-ml==0.5",
+        "h5py==2.10.0",
+        "ipython",
```

Comment:
@Philoso-Fish do we really need ipyhon btw? That seems to be more for notebooks etc?